pkg/subsystem/lists/linux_upstream/subsystems.go

```diff
@@ -0,0 +1,4619 @@
+// AUTOGENERATED FILE
```

Comment:
This needs the standard caption:
https://github.com/golang/go/issues/13560
https://go.dev/blog/generate
It's recognized by all Go tooling.